src/containers/AboutUs/index.js

```diff
@@ -0,0 +1,268 @@
+        </HookedParallax>
+        <HookedParallax style={{ position: 'absolute', bottom: -50, right: 150 }} multiplierY={1.5}>
+          <Square size={100} rotate={-22} color={props.theme.brandOrange} />
+        </HookedParallax>
+      </FounderContainer>
+      <JoinUsContainer>
+        <JoinUsTextContainer>
+          <AppText color={props.theme.brandDarkRed} fontSize="42" fontWeight="bold">
+            Want to join us?
+          </AppText>
+          <AppText color={props.theme.whiteColor} fontSize="32">
+            Do not hesitate to contact us!
+          </AppText>
+          <AppText color={props.theme.whiteColor} fontSize="22">
+            We are a small team that is always looking forward to grow. We are fun and very
+            dedicated.
+          </AppText>
+        </JoinUsTextContainer>
+      </JoinUsContainer>
+      <OurProcessContainer />
+    </Container>
+  );
+};
+
+// Components
+const Container = styled.div`
+  color: ${props => props.theme.lightDarkColor};
+  transition: all ease 200ms;
+`;
+
+const HeaderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 74vh;
+  width: 800px;
+  margin-left: 120px;
+  margin-top: 20vh;
+  overflow: hidden;
+`;
+
+const HeaderStyle = styled.div`
+  display: flex;
+  background-color: ${props => props.backgroundColor || props.theme.lightDarkColor};
+  color: ${props => props.theme.primary};
+`;
+
+const AppText = styled.p`
+  font-size: ${props => `${props.fontSize}px`};
+  color: ${props => props.color};
+  font-weight: ${props => props.fontWeight};
+  margin: 0;
+`;
+
+const SpanText = styled.span`
+  font-size: ${props => `${props.fontSize}px`};
+  color: ${props => props.color};
+  font-weight: ${props => props.fontWeight};
+`;
+
+const GradientButton = styled(Button)`
+  && {
+    background: ${props => `linear-gradient(45deg, ${props.color1} 30%, ${props.color2} 200%);`};
+    border-radius: 50px;
+    border: 0;
+    color: ${props => props.theme.brandWhite};
+    height: 48px;
+    font-weight: bold;
+    width: 400px;
+    margin-top: 20px;
+  }
+`;
+
+const Circle = styled.div`
+  display: flex;
+  height: ${props => `${props.size}px`};
+  width: ${props => `${props.size}px`};
+  border-radius: 50%;
+  background-color: ${props => props.color};
+  justify-content: center;
+  align-items: center;
+  .scroll-down {
+    width: 80px;
+    height: 80px;
+    padding: 0 0 24px;
+    cursor: pointer;
+    pointer-events: auto;
+    color: white;
+    right: ${props => `${props.size / 5}px`};
+    bottom: ${props => `${props.size / 6}px`};
+    position: relative;
+  }
+`;
+
+const Square = styled.div`
+  height: ${props => `${props.size}px`};
+  width: ${props => `${props.size}px`};
+  border-radius: 30px;
+  transform: ${props => `rotateY(0deg) rotate(${props.rotate}deg)`};
+  background-color: ${props => props.color};
+  opacity: 0.7;
+`;
+
+const FounderContainer = styled.div`
+  height: 100vh;
```

Comment:
There's a CSS bug when using `100vh` on mobile, it's better to use `window.innerHeight`, check out the homepage, specifically the `Container` component to see how I'm handling this over there.